spec.html

```diff
@@ -37183,7 +37183,7 @@
 
         <!-- es6num="B.3.4" -->
         <emu-annex id="sec-functiondeclarations-in-ifstatement-statement-clauses">
-          <h1>FunctionDeclarations in IfStatement Statement Clauses</h1>
+          <h1>FunctionDeclaration in IfStatement Statement Clauses</h1>
```

Comment:
This change is also incorrect. In step 1.a.ii.1 of [B.3.3](http://tc39.github.io/ecma262/#sec-block-level-function-declarations-web-legacy-compatibility-semantics):

> NOTE A var binding for F is only instantiated here if it is neither a VarDeclaredName, the
name of a formal parameter, or another FunctionDeclarations.

Should be:
> NOTE A var binding for F is only instantiated here if it is neither a VarDeclaredName, the
name of a formal parameter, or another FunctionDeclaration.